docs/android/emulator/hardware-acceleration.md

```diff
@@ -0,0 +1,233 @@
+---
+title: "Enable hardware acceleration for the Android emulator (Hyper-V & HAXM)"
+description: "When developing a .NET MAUI app, you'll want to use your computer's hardware acceleration features to maximize Android emulator performance."
+ms.date: 12/02/2021
+---
+
+# Hardware acceleration for better emulator performance (Hyper-V & HAXM)
+
+This article explains how to use your computer's hardware acceleration features to maximize Android emulator performance.
+
+With Visual Studio, you can easily test and debug your .NET MAUI app for Android in situations where an Android device isn't available. However, if hardware acceleration isn't available or enabled, the emulator will run too slow. You can drastically improve the performance of the emulator by enabling hardware acceleration and using special x86 virtual device images.
+
+<!-- TODO: What are the virtualization extensions on the new M1 ARM processors for Mac?
+           Also, isn't Hypervisor.Framework a macOS-only tech? Why is this at the start?
```

Comment:
Yes. It'll be at the start because the original doc will date back to when Mac support was more important than Windows support (it used to be the case that Mac support ruled, while Windows support lagged behind).